specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -835,6 +835,55 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/addNodeUser": {
+    "post": {
+      "tags": [
+        "MachineLearningComputes"
+      ],
+      "operationId": "MachineLearningCompute_AddNodeUser",
+      "description": "Adds user to the specified list of nodes in the cluster.",
+      "x-ms-examples": {
+        "AML Compute Add Node User": { "$ref": "./examples/amlComputeAddNodeUser.json" }
+      },
+      "parameters": [
+        {
+          "$ref": "#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/WorkspaceNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ComputeNameParameter"
+        },
+        {
+          "$ref": "#/parameters/APIVersionParameter"
+        },
+        {
+          "$ref": "#/parameters/NodeUserAccountCreateParameter"
```

Comment:
>tCreate [](start = 46, length = 7)

nit : Create => Add

I hope we are not creating the user as a side effect of this call.. If so, we need to model this as separate resource and add resource CRUD